lib/collection/src/collection_manager/optimizers/config_mismatch_optimizer.rs

```diff
@@ -59,6 +59,16 @@ impl ConfigMismatchOptimizer {
             .and_then(|vector_params| vector_params.on_disk)
     }
 
+    /// Check if current configuration requires sparse vectors index to be stored on disk
+    fn check_if_sparse_vectors_on_disk(&self, vector_name: &str) -> Option<bool> {
```

Comment:
Nitpick, this naming seems to be about storage, I propose to rename it `check_if_sparse_vector_index_on_disk`